### ISSUE TITLE
Implement the frontend part of TaskProfile 

### ISSUE THREAD

**Issue by sayan404** [sayan404/TaskPulse#1]
### More details
- this page will show three columns that will contain three sections to-do , doing , done .
- to-do will contain the pending tasks 
- doing section will contain the task that i am doing 
- done section will contain the task that i have completed 
- every task card will contain edit delete option and only cards of to-do section user can add or create tasks that will be directly added to the to-do section
- the cards can be drag and drop able  from one to another section
### Grab the issue
- to grab the issue comment below `I want to contribute in this issue under JWoC. Assign me this issue.`
### Contributor Guideline
- Read the README.md file before contributing

**Comment by JavidSumra** [comment]
I want to contribute in this issue under JWoC. Assign me this issue.

**Comment by itsmeabhishek03** [comment]
I can work on under JWOC

**Comment by sayan404** [comment]
I am assigning this issue. Happy coding 🚀.

**Comment by Kshitiz-monkecoder** [comment]
I want to contribute in this issue under JWoC. Assign me this issue.

**Comment by sayan404** [comment]
this issue is already assigned . can you raise some other issue in this repository . @Kshitiz-monkecoder

**Comment by sayan404** [comment]
HI @itsmeabhishek03 . Any progress in this issue ?

**Comment by itsmeabhishek03** [comment]
> HI @itsmeabhishek03 . Any progress in this issue ?

https://www.awesomescreenshot.com/video/24157047?key=4479081c03cca58655040220b62d3841

hey @sayan404 this is my progress till now. Features, I have to work on or modify - 
1) Work on ui/ux
2) Deal with drag and drop logic as you have seen it is not working properly 
Please share your opinion if any, Thank you

**Comment by sayan404** [comment]
Yeah, just watched the preview video, and it's working well. However, some modifications are needed. Just focus on the drag-and-drop functionality. The cards should be draggable between sections, both horizontally and vertically. Don't address styling for now. Later I'll create another issue for styling. Fix the drag-and-drop functionality and create a PR. @itsmeabhishek03

**Comment by itsmeabhishek03** [comment]
> Yeah, just watched the preview video, and it's working well. However, some modifications are needed. Just focus on the drag-and-drop functionality. The cards should be draggable between sections, both horizontally and vertically. Don't address styling for now. Later I'll create another issue for styling. Fix the drag-and-drop functionality and create a PR. @itsmeabhishek03

Working on it

**Comment by Hacxmr** [comment]
Hey! @sayan404 I want to contribute in this issue under JWoC. Assign me this issue.

**Comment by sayan404** [comment]
this issue is already assigned . Raise some other issue or stay tuned I will raise some other issues. @Hacxmr

**Comment by JavidSumra** [comment]
I want to contribute in this issue under JWoC. Assign me this issue.
Hey @sayan404 Sir I think Time Duration to Complete this issue is over so can you please assign it to me?

**Comment by sayan404** [comment]
Resolving certain issues can take more time than we initially thought. So let's wait for some more time. @JavidSumra

**Comment by sayan404** [comment]
HI @itsmeabhishek03  . Any update in this issue ?

**Comment by itsmeabhishek03** [comment]
> HI @itsmeabhishek03 . Any update in this issue ?

hey @sayan404 i am facing errors while implementing drag and drop functionality i am using react-dnd for it. i don't know where it's going wrong will try to raise PR ASAP.

**Comment by sayan404** [comment]
Okay if possible, make a PR on whatever you have done till now. @itsmeabhishek03 .

**Comment by JavidSumra** [comment]
Hey @sayan404 Please Check PR

**Comment by sayan404** [comment]
Anyone who wants to work more on this issue can comment below.

**Comment by JavidSumra** [comment]
Hey @sayan404 I can create the more this type of API

**Comment by sayan404** [comment]
okay @JavidSumra  raise issues on that.
